### PR TITLE
Add nextgen picture to .gitattributes

### LINKED ISSUE #16713: Add git support for next gen picture.
### Description

Currently .gitattributes not support nextgen (animated) pictures.
- webp
- avif 

It can lead to issue as they can be seen as TXT file and lead to EOF insertion.  

### CakePHP Version

4.x

### PR DESCRIPTION
**Pull request by HumanG33k**
Add nextgen picture to .gitattributes in order to prevent git commit/push EOF issue. resolve cakephp/app#906

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -11,6 +11,8 @@
 *.pem eol=lf
 
 # Denote all files that are truly binary and should not be modified.
+*.avif binary
+*.webp binary
 *.png binary
```
Comment (ADmad) [inline .gitattributes:16]: The core doesn't these type of files so there isn't much value in this change. We can instead add avif to the .gitattributes of cakephp/app repo (it already has webp).

Comment (VoidScute) [reply .gitattributes:16]: humm strange because i have issues with webp last week (but on a v3 -> v4 migrated app).
Do you want i make the push on app ?

Comment (markstory) [reply .gitattributes:16]: >  Do you want i make the push on app ?

Yes please, that is how we push updates to the application skeleton code. New users generally use the latest app skeleton and existing applications can overlay the new files on theirs to selectively apply changes in the default skeleton.

**Comment by othercorey** [comment]
Moved to cakephp/app.

**Pull request closed without merging**